app/views/users/_form.html.erb

```diff
@@ -0,0 +1,53 @@
+<%= form_for(@user, :html => {class: "form col justify-content-center"}) do |f| %>
+  <% @error = f.object.errors %>
+  <div class="col justify-content-center">
+    <p><%= @error.full_messages_for(:name).join(", ") %></p>
+    <%= f.text_field :name, class: "form-control",
+      placeholder: t(".place_name") %>
+
+    <p><%= @error.full_messages_for(:email).join(", ") %></p>
+    <%= f.email_field :email, class: "form-control",
+      placeholder: t(".place_email") %>
+
+    <p><%= @error.full_messages_for(:password)[0] %></p>
+    <%= f.password_field :password, class: "form-control",
+      placeholder: t(".place_password") %>
+
+    <p><%= @error.full_messages_for(:password_confirmation)[0] %></p>
+    <%= f.password_field :password_confirmation, class: "form-control",
+      placeholder: t(".place_password_repeate") %>
+
+    <div class="col">
+      <%= f.radio_button :gender, User.genders.key(0), class: "col-sm-1" %>
```

Comment:
` User.genders.key(0)` tào lao, code như bên dưới, chứ chú setting 0/1... thì cũng như code ban đầu của chú rồi
` User.genders.key(User.genders.male)`

Reply:
:D

Reply:
"male is undefined" chi oi  :3

Reply:
User.genders.key(User.genders[:male]) nhu nay nha chi?

Reply:
mà như này thì em lại thấy giống:
f.radio_button :gender, :male

Reply:
giống nhưng 1 cái là em setting cứng, ko cái là lấy theo giá trị enum mà

Reply:
User.genders.key(User.genders[:male])
em thấy cái này cũng là truyền vào 1 chuổi cứng là [:male] để đến ENUM lấy key. Sau đó là lấy key để truyền lại vào ENUM để lấy chuổi...
Sau request rồi lại gửi chuổi lên lại ENUM để lấy key để lưu vào db.
Còn...
:male nó cũng là truyền lên chuổi rồi lại đến enum để lấy key lưu vào db
2 cái này cũng là đưa chuổi cứng đến ENUM và lấy key thôi mà chị?